collectors/timex.plugin/plugin_timex.c

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "plugin_timex.h"
+#include "sys/timex.h"
```

Comment:
can this be a problem on non linux systems? should we compile this code only on linux?

Reply:
FreeBSD also supports `timex`,  I am not sure about Mac.

Reply:
yeah, i found it http://fxr.watson.org/fxr/source/sys/timex.h

Linux's adjtimex(2) is called ntp_adjtime(2) in FreeBSD.

My point is that perhaps we need to update make instructions to not build this collector on non linux systems for now. Not a specialist, just highlighting a possible problem.

Reply:
Please look at https://github.com/netdata/netdata/blob/40e247375b6bc64399a8455a89c44cbb66767e04/Makefile.am#L717-L721

Reply:
Cool 😄